include/deal.II/numerics/vector_tools_copy_to_different_dof_handler.h

```diff
@@ -0,0 +1,335 @@
+/*
+ * copy_to_different_dof_handler.h
+ *
+ *  Created on: Apr 29, 2020
+ *      Author: mwichro
+ */
+
+#ifndef INCLUDE_INTERPOLATE_TO_DIFFERENT_DOF_HANDLER_H_
+#define INCLUDE_INTERPOLATE_TO_DIFFERENT_DOF_HANDLER_H_
+
+#include <deal.II/base/quadrature.h>
+
+#include <deal.II/dofs/dof_handler.h>
+
+#include <deal.II/lac/la_parallel_vector.h>
+
+// apply_transform is defined here
+// I am probably including wrong file, so please check this
+#include <deal.II/numerics/vector_tools_interpolate.templates.h>
+
+namespace dealii
+{
+  namespace VectorTools
+  {
+    // Modyfied internal interpolate
+    //
```

Comment:
Can you copy the documentation here, modifying what is different from the other version?